src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -45,20 +43,23 @@ public OtpUserController(String apiPrefix) {
 
     @Override
     OtpUser preCreateHook(OtpUser user, Request req) {
-        // Check API key and assign user to appropriate third-party application. Note: this is only relevant for
-        // instances of otp-middleware running behind API Gateway.
-        String apiKey = req.headers("x-api-key");
-        ApiUser apiUser = Persistence.apiUsers.getOneFiltered(Filters.eq("apiKeys.value", apiKey));
-        if (apiUser != null) {
-            // If API user found, assign to new OTP user.
-            user.applicationId = apiUser.id;
-        } else {
-            // If API user not found, report to Bugsnag for further investigation.
-            BugsnagReporter.reportErrorToBugsnag(
-                "OTP user created with API key that is not linked to any API user",
-                apiKey,
-                new IllegalArgumentException("API key not linked to API user.")
-            );
+        RequestingUser requestingUser = Auth0Connection.getUserFromRequest(req);
+        // If an Api user is present it is assumed an attempt is being made to create an OTP user.
```

Comment:
This comment doesn't make sense to me: `it is assumed an attempt is being made to create an OTP user`. That's already implied by the fact that they have made a POST request to the createUser endpoint.

Reply:
I've removed it.